backend/corpora/common/entities/collection.py

```diff
@@ -174,14 +174,19 @@ def list_collections_curation(
         resp_collections = []
         for collection in session.query(cls.table).filter(*filters).all():
             # Add a Collection-level processing status
-            status = ProcessingStatus.SUCCESS
+            status = None
+            has_statuses = False
             for dataset in collection.datasets:
                 processing_status = dataset.processing_status
-                if processing_status.processing_status == ProcessingStatus.PENDING:
-                    status = ProcessingStatus.PENDING
-                elif processing_status.processing_status == ProcessingStatus.FAILURE:
-                    status = ProcessingStatus.FAILURE
-                    break
+                if processing_status:
+                    has_statuses = True
+                    if processing_status.processing_status == ProcessingStatus.PENDING:
+                        status = ProcessingStatus.PENDING
+                    elif processing_status.processing_status == ProcessingStatus.FAILURE:
+                        status = ProcessingStatus.FAILURE
+                        break
+            if has_statuses and not status:  # At least one dataset processing status exists, and all were SUCCESS
```

Comment:
in the situation where there are no dataset processing statuses, collection-level status would be None. Just want to confirm--is that the expected behavior?